services/black/src/commands/Validation/SetGuestInformationValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace artiso.AdsdHotel.Black.Commands.Validation
+{
+    public static class SetGuestInformationValidator
+    {
+        public static bool IsValid(SetGuestInformation sgi)
+        {
+            if (string.IsNullOrEmpty(sgi.GuestInformation.FirstName))
```

Comment:
Suggestion: check for null, even if it's not nullable.
Somebody evil could do:
```csharp
SetGuestInformation? foo = null;
SetGuestInformationValidator.IsValid(foo!);
```

Reply:
That contradicts the purpose of NRTs. If someone does that, he should expect an exception ;-)